src/mixins/index.ts

```diff
@@ -11,11 +9,12 @@ import ResourceLookupMixin from './resource-lookup-mixin'
 export {
   AllowableActionsMixin,
   CommonMixin,
-  CountriesProvincesMixin,
-  DateMixin,
   DirectorMixin,
   EnumMixin,
   FilingMixin,
   NameRequestMixin,
   ResourceLookupMixin
 }
+
+export { default as DateMixin } from './date-mixin'
+export { CountriesProvincesMixin } from '@bcrs-shared-components/mixins'
```

Comment:
This is what I mentioned above in `package.json`. The mixin now is imported from the shared repo to fix the `require` problem.